lib/mintacoin_web/views/accounts_view.ex

```diff
@@ -0,0 +1,23 @@
+defmodule MintacoinWeb.AccountsView do
+  @moduledoc """
+  This module contains the JSON response for the account's endpoints
+  """
+
+  use MintacoinWeb, :view
+
+  alias Mintacoin.Account
+
+  @type template :: String.t()
+  @type assigns :: map()
+  @type json :: map()
+
+  @spec render(template :: template(), assigns :: assigns()) :: json()
+  def render("account.json", %{resource: %Account{} = account}),
+    do: %{status: 201, data: account}
```

Comment:
@jhonatan-kmt @FelipeGuzmanSierra I saw that you added a `Jason Encoder` to the account scheme, but IMO, to avoid that Encoder, and to have a better understanding of what we are returning to the endpoint, I think we should do a pattern matching here like so:
```suggestion
  def render("account.json", %{resource: %Account{address: address, signature: signature, seed_words: seed_words}}),
    do: %{status: 201, data: %{address: address, signature: signature, seed_words: seed_words}}
```
What do you think?